unified-runtime/source/adapters/level_zero/v2/common.hpp

```diff
@@ -79,12 +79,15 @@ struct ze_handle_wrapper {
       return;
     }
 
-    if ((ownZeHandle && !IsInteropNativeHandle) ||
```

Comment:
can we remove `IsInteropNativeHandle` member now?

Reply:
with this more robust handling, yes, I can remove that if we feel we never need that info.

I can add that change.

Reply:
Fixed in the latest commit

Reply:
Thanks! Do you think we can also remove the one from source/adapters/level_zero/common.hpp and source/adapters/level_zero_v2/common.hpp? (the latter one seems to never be used actually)

Reply:
I think I removed it in all places, can you confirm?

Reply:
I realized I forgot to post the removal I did in a previous commit locally, I will update shortly.

Reply:
fixed in the latest patch